src/modules/flow/lcd/Kconfig

```diff
@@ -0,0 +1,4 @@
+config FLOW_NODE_TYPE_LCD
+	tristate "Node type: lcd"
+	depends on USE_I2C
```

Comment:
this is going to blow, be more specific, "lcd" is not enough as a name for this module as it covers a single kind of lcd.